CMSIS/CoreValidation/Source/CV_CoreFunc.c

```diff
@@ -32,6 +32,12 @@ static void TC_CoreFunc_IPSR_IRQHandler(void) {
   irqXPSR = __get_xPSR();
 }
 
+#if defined(__ti__)
```

Comment:
This is only test code, so performance is not big issue.
Hence, I'd add `__NOP` in any case and remove compiler dependency here.